nodl_python/nodl/_parsing/_v1/_parsing.py

```diff
@@ -30,9 +30,22 @@ def _parse_action(element: etree._Element) -> Action:
     if not (server or client):
         raise errors.AmbiguousActionInterfaceError(element)
 
-    policy = _parse_qos(element.find('qos'))
+    # List of names of possible elements in the xml doc
+    profile_keys = (
+        'goal_service_qos_profile',
+        'result_service_qos_profile',
+        'cancel_service_qos_profile',
+        'feedback_sub_qos_profile',
+        'status_sub_qos_profile',
```

Comment:
Can you add where you're getting these to the comment, please?

Reply:
a558e9a